torch/csrc/jit/mobile/function.cpp

```diff
@@ -33,7 +33,7 @@ void Function::append_instruction(OpCode op, int X, int N) {
 bool Function::append_operator(
     const std::string& name,
     const std::string& overload_name,
-    int64_t model_version) {
+    int64_t num_specified_args) {
```

Comment:
This change seems like it could go unnoticed. Does it make sense to add one more argument and see what breaks? If we add and remove an `int64_t` argument, then existing call sites will still build successfully (in case we missed fixing some of them up).

Reply:
It is a replacement for a field with the same semantics but a clearer name.

It's a valid concern, but definitely we should't changing the actual committed code's signature just to verify we didn't miss refactoring somewhere. I would expect @iseeyuan already checked all necessary places (probably doing what you suggest).

Reply:
The only call site of this function is in mobile/import.cpp. Also I think it's a good decoupling from model version. The model version of 3 check is too special.

Reply:
@raziel Yep! I meant locally (not for checking in this change) - thanks for clarifying that!